### ISSUE TITLE
Needed option to disable dark mode for the Lazarus designer window

### ISSUE THREAD

**Issue by furious-programming** [zamtmn/metadarkstyle#1]
If possible, please add the ability to disable dark mode for the designer so that it is compatible with the system theme. Thanks to this, the designer will always present the final appearance of the window, i.e. it will be a real WYSIWG, despite the dark mode turned on in the IDE.

**Comment by zamtmn** [comment]
no. I haven't figured out this issue yet. Fortunately, I don't use the designer in my work))

**Comment by flowCRANE** [comment]
@zamtmn any plans to implement this?

**Comment by zamtmn** [comment]
Unfortunately, this cannot be done - theme works globally, redefining system colors and draw methods

**Comment by flowCRANE** [comment]
So how does this package's code render windows' contents if it has no control over what color palette is used?

**Comment by flowCRANE** [comment]
If so, then there is probably no reason why the feature mentioned in this issue cannot be implemented. All you need to do is check the parent of the control and if it is a designer window, use the color palette for the light theme.

In general, it would be good if this package allowed registering forms and assigning them dedicated color palettes (e.g. a dictionary with window-palette pairs). In this way, this package would be flexible, without hard-coded and simplified mechanisms. You could use something like this creatively — e.g. have the main window in a light (white) theme, have a red palette for the error message window, a yellow palette for the warning window, etc. People would be delighted to have something like this at their disposal, and especially if you could modify colors at runtime and see the changes immediately in the window, without having to restart the application.

I know that implementing something like this would require a lot of work and time, but IMO it is worth looking into this topic.

**Comment by zamtmn** [comment]
No, unfortunately this is not possible.
